showcase/src/app/home/home.component.html

```diff
@@ -1,100 +1,114 @@
-<h1 class="mat-display-2" translate>Stark documentation</h1>
+<div class="about-background">
+	<h1 class="about-title" [innerHTML]= "'SHOWCASE.HOMEPAGE.MAIN_TITLE' | translate"></h1>
+</div>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_BEFORE_DETAIL' | translate">
+</h2>
+<h4 [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_DETAIL' | translate"></h4>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_AFTER_DETAIL' | translate">
+</h2>
 
-<div fxLayout="column" fxLayout.gt-xs="row wrap" fxLayoutAlign.gt-md="space-between center" fxLayoutAlign.md="space-around center"
- fxLayoutAlign.lt-md="center center" fxLayoutGap.lt-lg="20px">
-	<a href="https://stark.nbb.be/api-docs/stark-core/latest">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark Core - Latest</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/logo-blue.svg">
-			</mat-card-content>
-		</mat-card>
-	</a>
-	<a href="https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-core">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark Core</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/core.png">
-			</mat-card-content>
-		</mat-card>
-	</a>
-	<a href="https://stark.nbb.be/api-docs/stark-ui/latest">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark UI - Latest</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/logo-blue.svg">
-			</mat-card-content>
-		</mat-card>
-	</a>
-	<a href="https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-ui">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark UI</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/logo-ui.png">
-			</mat-card-content>
-		</mat-card>
-	</a>
+<section class="stark-section bottom-margin">
+	<mat-accordion>
+		<mat-expansion-panel>
+			<mat-expansion-panel-header>
+				<mat-panel-title [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_NEWS_TITLE' | translate">
```

Comment:
The titles of the collapsibles are not aligned vertically :(
![not centered collapsible title](https://user-images.githubusercontent.com/5698318/46203450-12fdf780-c31a-11e8-8805-81494c91c811.PNG)

Reply:
`<mat-panel-title>` add an extra margin, we should maybe remove it in the collapsible component. @christophercr what do you think?
Other solution would be to use flexbox in the header to align it